playwright.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
     reuseExistingServer: !process.env.CI,
     stdout: "pipe",
     stderr: "pipe",
-    timeout: 10000,
+    timeout: 30000,
```

Comment:
This is just the timeout for the server to start, i dont think weve ever had a failure here